src/components/Profile/MyPosts/MyPosts.jsx

```diff
@@ -15,11 +15,12 @@ const MyPosts = (props) => {
   const creatPost = () => {
     const text = newPostElement.current.value;
     addPost(text);
+    newPostElement.current.value = ' ';
```

Comment:
So, you're basically setting the value of the input to be 1 space - which is not empty.
Is it intended? Maybe it's better to set this value to empty string '' instead of 1 space ' '?

Reply:
Там должна быть пустая строка. Исправлено.